test/Microsoft.TemplateEngine.TestHelper/PackageManager.cs

```diff
@@ -118,11 +119,13 @@ public string PackNuGetPackage(string projectPath)
                         $"{Environment.NewLine}StdErr: {stdErr}.");
```

Comment:
We should add info about the exit code here as well - I can see cases where pack seemingly succeeded, but returned nonzero exit code - e.g.: https://dev.azure.com/dnceng-public/public/_build/results?buildId=34308&view=ms.vss-test-web.build-test-results-tab&runId=683860&resultId=100021&paneView=debug

More info will be helpful in such a case. It might be related to unresolved issue: https://github.com/dotnet/sdk/issues/12298